core/common/src/main/java/alluxio/underfs/Fingerprint.java

```diff
@@ -33,6 +33,11 @@
   /** These tags are optional, and are serialized after the required tags. */
   private static final Tag[] OPTIONAL_TAGS = {Tag.CONTENT_HASH};
 
+  /** These tags are all the metadata tags in the fingerprints. */
+  private static final Tag[] METADATA_TAGS = {Tag.OWNER, Tag.GROUP, Tag.MODE};
+  /** These tags are all the metadata tags in the fingerprints. */
```

Comment:
I think this comment needs to be updated?

Also, I think the comment on line 30 can be removed.